deploy/overlays/webhook/webhook_config.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
 webhooks:
   - name: binding.spo.io
     failurePolicy: Fail
+    timeoutSeconds: 5
```

Comment:
If this is realistically something that deployers want configurable, why not make it configurable? (What if your policy will say no longer than 4 seconds next year)?

Reply:
Happy to make it configurable. How do you suggest to do that in this situation?

Reply:
ugh, I'm sorry, only now did I notice that this is the overlay, for some reason I thought you were modifying the go sources in bindata with a hardcoded config. Sorry, going to ack this now.

Reply:
Thanks for reviewing it!